server/views/taskList/caSubmitRefusalTask.pug

```diff
@@ -0,0 +1,22 @@
+- var canSendToDm = licenceStatus.stage === 'ELIGIBILITY' || allowedTransitions.caToDm
+
```

Comment:
i don't think you can reuse allowedTransitions.caToDm, certainly _shouldn't_ reuse it

should be caToDmRefusal or something